### ISSUE TITLE
Just Dance on PC

### ISSUE THREAD

**Issue by Bartek5347** [redphx/joydance#57]
I know this script is almost 2 years old, but I am curious if it still works. I tried to run dance.py but it doesn't work. 
I tried to use help from #53 but still nothing...

**Comment by BroJac5246** [comment]
Yes, it still works. I was able to get it to work on Windows 11 relatively easily, but I did need to mess with pip a little bit. Basically, I think some of the older versions of some dependencies don't like newer Python versions, but you can update them and they'll still work with JoyDance. 

I'm willing to help you but I'm away this weekend so I can't look at exactly what I did yet.

**Comment by Bartek5347** [comment]
oh okay, i'll try that thank you <3

**Comment by BroJac5246** [comment]
> oh okay, i'll try that thank you <3

Btw I used Python 3.11 from the Microsoft Store so I know that version works if you're having trouble

**Comment by BroJac5246** [comment]
@Bartek5347 were you able to get it to work?

**Comment by Blasetvrt** [comment]
> @Bartek5347 were you able to get it to work?

Also having trouble here. Mine won't even load up the web UI, even tho there didn't seem to have any problem installing dependencies. Using 3.10.11 here tho.

**Comment by Bartek5347** [comment]
I changed import aio to import aiohttp in dance.py i think

**Comment by Blasetvrtumi** [comment]
Yeah, I should've noticed it was not a python issue but a JS one, my issue was the black screen one, which you can only notice by checking the devtools console to see if it has issues finding the modules, so just chsnging the ecmascript import format worked.

**Comment by BroJac5246** [comment]
Were both of you able to get it to work?
@Blasetvrt @Bartek5347
